src/measure_theory/measure_space.lean

```diff
@@ -1428,6 +1428,20 @@ ennreal.coe_to_nnreal (measure_ne_top μ univ)
 
 instance finite_measure_zero : finite_measure (0 : measure α) := ⟨by simp⟩
 
+instance finite_measure_add [finite_measure μ] [finite_measure ν] : finite_measure (μ + ν) :=
+{ measure_univ_lt_top :=
+  begin
+    rw [measure.coe_add, pi.add_apply, ennreal.add_lt_top],
+    exact ⟨measure_lt_top _ _, measure_lt_top _ _⟩,
+  end }
+
+instance finite_measure_smul_nnreal [finite_measure μ] {r : ℝ≥0} : finite_measure (r • μ) :=
+{ measure_univ_lt_top :=
+  begin
+    erw [measure.coe_smul, pi.smul_apply, algebra.id.smul_eq_mul, ennreal.coe_of_nnreal_hom],
```

Comment:
Which lemma needs `erw` instead of `rw`?

Reply:
`coe_smul` needs `erw` since its defined with `ennreal` instead of `nnreal`

Reply:
I think `coe_smul` should be renamed to `coe_ennreal_smul`, and then this lemma added:
```
@[simp, norm_cast] theorem coe_smul {R} [is_scalar_tower R (ℝ≥0∞) (measure α)] (c : R) (μ : measure α) : ⇑(c • μ) = c • μ :=
sorry
```
which ought to be easy to prove by rewriting `c • μ` into `(c • 1 : ℝ≥0∞) • μ`

Reply:
I think I'm missing something since I can't seem to prove that. Would you mind making a pr for this?

Reply:
Sure, no need to do that now. How about you just add a new lemma `coe_nnreal_smul` which is probably just proved via `coe_smul`, and I'll unify them in a follow up?

Reply:
Alright, I will do that early tommorow

Reply:
I've now added `coe_nnreal_smul` though it seems like I never needed the `rw` in the first place since it is defeq